app/assets/stylesheets/application.scss

```
@@ -99,7 +99,15 @@ form{
 }
 
 #todo_title {
-  color: white;
+  color: #F70A7C;
```

You should use sass variables here so you don't hardcode the color provided by the styleguide

```suggestion
  color: $magenta;
```